dist/index.html

```diff
@@ -17,17 +17,24 @@
       rel="stylesheet"
     />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <script defer src="bundle7b117f43c0ff9b47859b.js"></script></head>
+  <script defer src="bundle.59dda23859e677e29610.js"></script></head>
   <body>
     <header>
-      <nav-menu nav-items='["Collections", "Men", "Women", "About", "Contact"]'>
-        <a href="index.html" slot="nav-item" class="nav-link">Home</a>
-        <a href="collections" slot="nav-item" class="nav-link">Collections</a>
-        <a href="men" slot="nav-item" class="nav-link">Men</a>
-        <a href="women" slot="nav-item" class="nav-link">Women</a>
-        <a href="about" slot="nav-item" class="nav-link">About</a>
-        <a href="contact" slot="nav-item" class="nav-link">Contact</a>
-      </nav-menu>
+      <div class="menu-section">
+        <nav-menu></nav-menu>
+        <img src="assets/logo.svg" alt="Sneakers Logo" class="logo" />
+      </div>
+
+      <div class="cart-section">
+        <shopping-cart></shopping-cart>
+        <button class="profile-btn">
```

Comment:
Button is een goed voorbeeld om daar een web component van te maken. die kan je dan zo maken, dat je aan de hand van verschillende attributes, verschillende buttons kan tonen.